docs/components/store.md

```diff
@@ -399,7 +399,47 @@ Following options are used for metadata caching (meta.json files, deletion mark
 
 The yml structure for setting the in memory cache configs for caching bucket is the same as the [in-memory index cache](#in-memory-index-cache) and all the options to configure Caching Buket mentioned above can be used.
 
-Note that chunks and metadata cache is an experimental feature, and these fields may be renamed or removed completely in the future.
```

Comment:
I don't think it's experimental anymore. It's been in the code quite a while and no issues came up. Thus, I've removed this sentence.